### PR TITLE
Update rq041 code sets

### PR DESCRIPTION
**Pull request by Georgetilston**

### CODE REVIEW

File: shared/clinical-code-sets/medications/female-sex-hormones/1/female-sex-hormones.snomed.txt [hunk 1]
```diff
@@ -674,3 +674,57 @@
 36394711000001107	Progesterone 25mg/1.112ml solution for injection vials
 36394811000001104	Lubion 25mg/1.112ml solution for injection vials (Pharmasure Ltd)
 36395011000001109	Lubion 25mg/1.112ml solution for injection vials (Pharmasure Ltd)
+11391011000001107	Testosterone 300micrograms/24hours patches (product)
```
Comment (rw251) [inline shared/clinical-code-sets/medications/female-sex-hormones/1/female-sex-hormones.snomed.txt:677]: I don't think testosteron should be in this code set. If that's right then remove the ~15 instances. Also checkout the things like **Testo**derm, **Test**ogel, **Test**im as they sound a bit like testosterone products.

File: shared/clinical-code-sets/medications/statins/1/statins.snomed.txt [hunk 2]
```diff
@@ -0,0 +1,846 @@
...
+41548011000001105	Atorvastatin 40mg tablets (Brown & Burk UK Ltd)
+9316111000001109	Simvastatin 80mg / Ezetimibe 10mg tablets (product)
+9316011000001108	Simvastatin 40mg / Ezetimibe 10mg tablets (product)
+4172611000001100	Rosuvastatin 40mg tablets (product)
+4172511000001104	Rosuvastatin 20mg tablets (product)
+9315911000001100	Simvastatin 20mg / Ezetimibe 10mg tablets (product)
+4172411000001103	Rosuvastatin 10mg tablets (product)
+32170911000001100	Cholib 145mg/20mg tablets (Viatris UK Healthcare Ltd) (product)
+32234211000001101	Fenofibrate 145mg / Simvastatin 20mg tablets (product)
+42209911000001100	Fenofibrate micronised 160mg tablets (product)
```
Comment (rw251) [inline shared/clinical-code-sets/medications/statins/1/statins.snomed.txt:842]: Fenofibrate is not a static. Cholib (fenofibrate & simvastatin) is fine, but the ones that are just fenofibrate should be removed.

**Review by rw251 — COMMENTED** [review]
Just a few comments.

**Comment by Georgetilston** [comment]
thanks for spotting those errors. I've committed changes for them.

**Review by rw251 — APPROVED** [review]

**Pull request merged**